docs/fast_data/tutorials/low-code.mdx

```diff
@@ -26,348 +26,347 @@ To do that, in the console, you will:
-In this example, we have the registry_id in both the Registry Projection and in the Order Projection, so we need to configure the Projection Changes Schema to read the registry_id from the updated Projection.
+<details>
+  <summary> Orders Topics</summary>
+  <p>
 
-```json {4,7,9,13,15} showLineNumbers
-{
-  "version": "1.0.0",
-  "config": {
-    "order-history": {
-      "paths": [
-        {
-          "path": [ "order", "registry" ],
-          "identifier": {
-            "registry_id": "registry_id"
-          }
-        }
-      ]
-    }
-  }
-}
-```
+These are the suggested naming convention of topics defined in Kafka:
 
-With this configuration, the Single View that we are going to create can be updated with both of the following ways:
-- if we update the Projection _registry_, then we can read the registry_id directly from there.
-- if we update the Projection _order_, we will search the related document inside the Projection _registry_, then we can read the registry_id.
+* __Ingestion__: `demo.development.delivery.order.ingestion` 
+* __Pr-Update__: `demo.development.delivery.order.pr-update` 
 
-You can find a more thorough explanation of the _Projection Changes Schema_ on the [dedicated page](/fast_data/configuration/config_maps/projection_changes_schema.md) of the documentation.
+</p>
+</details>
 
-### Expose Projection Endpoints
+### Link to Microservice
 
-At this point, we have configured the microservice, we already have one System of Record and two projections, but we need to expose the Projection endpoints. It is not mandatory to expose the Projection endpoints to create, update and work with the Single View, but if you want to expose the raw data stored in the projection you need to create the endpoints.
+Before receiving messages from the ingestion topic defined in the projection, you have to link it to a SoR microservice. 
+
+Go back to the `Services` section of the `delivery` SoR, and select the microservice you have attached during the previous paragraph: there, on the `Projections` card, you can pick one or more projections.
+
+![Link Projection to SoR Microservice](img/35-link-pr-to-PS.png)
+
+In this particular case, both `registry` and `orders` will be linked to the same microservice.
+
+Going back to the projection's page, you can see that the microservice linked to it will appear under the `Microservice` card.
+
+## Release Projections
+
+At this point, we have configured the microservice, we already have one system of record and two projections, but we need to expose the Projection endpoints. 
+
+### Create Endpoints
+
+:::tip
+It is not mandatory to expose the Projection endpoints to create, update and work with the Single View, but if you want to expose the raw data stored in the projection you need to create the endpoints.
+:::
 
 In the Endpoint section, we can click on the **Create new endpoint** button to create the Projection's endpoints.
 ![Create Endpoint's Projection](img/13-create-endpoint.png)
```

Comment:
Update the image using the new navigation or if the navigation bar in the image haven't any particular meaning cut it.

Reply:
updated!